pkgs/development/python-modules/dbf/default.nix

```diff
@@ -19,10 +19,15 @@ buildPythonPackage rec {
     hash = "sha256-MFEi1U0RNvrfDtV4HpvPgKTCibAh76z7Gnmj32IubYw=";
   };
 
+  # Workaround for https://github.com/ethanfurman/dbf/issues/48
+  patches = lib.optional python.stdenv.isDarwin ./darwin.patch;
```

Comment:
I'm uncertain of this, but since you have the `skipIf` in the patch, would it be better if you applied the patch unconditionally?